ads/google/a4a/utils.js

```diff
@@ -92,6 +92,9 @@ export let NameframeExperimentConfig;
  */
 export const TRUNCATION_PARAM = {name: 'trunc', value: '1'};
 
+/** @const {Object} */
+const CDN_REGEXP = /^https:\/\/([a-zA-Z0-9_-]+\.)?cdn\.ampproject\.org/;
```

Comment:
We already have this regexp in ads/google/a4a/utils.js#isGoogleAdsA4AValidEnvironment however that also includes crypto check (which you don't want).  Can you refactor to use common regexp?